tools/klee/main.cpp

```diff
@@ -1151,6 +1151,10 @@ int main(int argc, char **argv, char **envp) {
   parseArguments(argc, argv);
   sys::PrintStackTraceOnErrorSignal(argv[0]);
 
+  if (EntryPoint == "") {
```

Comment:
To be consistent, I'd use `.empty()` here as in the watchdog check below.

Reply:
Fix